test/bdd/pkg/issuer/issuer_steps.go

```diff
@@ -249,6 +286,25 @@ func (e *Steps) sendCreateRequest(req []byte) (*docdid.Doc, error) {
 	return didDoc, nil
 }
 
+func checkDIDAvailable(did string) error {
```

Comment:
https://github.com/trustbloc/edge-service/blob/f10282efb00746eb3a639b63bc85fd45448ef77f/test/bdd/pkg/vc/vc_steps.go#L606

Reply:
duplicate code

Reply:
Thanks. I wasn't aware of that code piece.

Reply:
The function you mentioned is not exposed. I'll move this along with other common code used across BDD as part of #132 as this change is already close to ~500 lines.